cms/djangoapps/contentstore/views/course.py

```diff
@@ -514,6 +514,7 @@ def filter_ccx(course_access):
     staff_courses = UserBasedRole(request.user, CourseStaffRole.ROLE).courses_with_role()
     site_courses = UserBasedRole(request.user, GlobalCourseCreatorRole.ROLE).courses_with_role()
     all_courses = filter(filter_ccx, instructor_courses | staff_courses | site_courses)
+    all_courses = filter_courses_based_on_org(request, list(all_courses))
```

Comment:
we don't need to convert `all_courses` to list here
`all_courses = filter_courses_based_on_org(request, all_courses)`